docs/language_concepts/data_types/03_strings.md

```diff
@@ -41,3 +41,23 @@ fn main() {
     assert(message_bytes[0] == message_vec.get(0));
 }
 ```
+
+## Escape characters
+
+You can use escape characters for your strings, as much as escape sequences:
```

Comment:
I think this phrasing is confusing. Can you just remove "as much as escape sequences"?